### DISCUSSION TITLE
Is GHES enabled with  docsify?

### DISCUSSION THREAD

**Discussion by nair-sumesh** [community/community#85666] [Enterprise]
### Select Topic Area

Question

### Body

Hi,

Is [docsify](https://github.com/docsifyjs/docsify) available in GHES?
If yes, since which version?
if not, is there any roadmap to bring it to GHES?

Regards,
SN

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @nair-sumesh ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by nair-sumesh** [comment]
Anyone here?
